subsys/net/lib/nrf_cloud/src/nrf_cloud_info.c

```diff
@@ -140,25 +141,30 @@ int nrf_cloud_print_details(void)
 	LOG_INF("Sec tag:   %d", nrf_cloud_sec_tag_get());
 	LOG_INF("Host name: %s", host_name);
 
-	/* Tenant will not be known unless device is connected to cloud with MQTT. */
+#endif /* CONFIG_NRF_CLOUD_VERBOSE_DETAILS */
+
+	return err;
+#else
+	return 0;
+#endif /* CONFIG_NRF_CLOUD_PRINT_DETAILS */
+}
+
+int nrf_cloud_print_cloud_details(void)
+{
+	int err = 0;
+#if defined(CONFIG_NRF_CLOUD_VERBOSE_DETAILS)
+
 #if defined(CONFIG_NRF_CLOUD_MQTT)
-	char stage[NRF_CLOUD_STAGE_ID_MAX_LEN];
 	char tenant[NRF_CLOUD_TENANT_ID_MAX_LEN];
 
-	err = nct_stage_get(stage, sizeof(stage));
-	if (!err) {
-		LOG_INF("Stage:     %s", stage);
-	} else {
-		LOG_ERR("Error determining nRF Cloud stage: %d", err);
-	}
```

Comment:
The commit description says this logic got separated out, but it seems to just be removed?

Reply:
Yes, all that is left is printing the team ID.

Justin suggested it because the stage is in the host name already, and customers don't use stages anyway.

Reply:
Ah, fair enough